app/assets/javascripts/components/school/school_comment_create.es6.jsx

```diff
@@ -0,0 +1,93 @@
+class SchoolCommentCreate extends Component {
```

Comment:
Since this probably parallels `StudentCommentCreate`, we might create a `template` component `CommentCreate` to extend from in the future (refactoring).